### ISSUE TITLE
Error 'Unknown Expression' when parsing interface/class extending other types using property access

### ISSUE THREAD

**Issue by panjiesw** [Swatinem/rollup-plugin-dts#10]
I've found another use case that's either a bug or just not yet covered by this plugin.

When trying to parse an interface/class extending other interface/class like this (common in React use case)

```tsx
import React from 'react';

export interface MyComponentProps extends React.HtmlHTMLAttributes<HTMLDivElement> {
  foo: string;
}

export class MyComponent extends React.Component<MyComponentProps> {}
```

It produced error like this:

```
{ kind: 189, code: ' React.HtmlHTMLAttributes' }
[!] Error: Could not load <rootProject>/src/index.ts: Unknown Expression
```

It also happened in `React.Component` node. 

Seems like heritage clauses parsing still doesn't cover property access expression like that.

**Comment by panjiesw** [comment]
If I change the code to extends without `React` namespace

```tsx
import {HtmlHTMLAttributes, Component} from 'react';

export interface MyComponentProps extends HtmlHTMLAttributes<HTMLDivElement> {
  foo: string;
}

export class MyComponent extends Component<MyComponentProps> {}
```

Then it produced error like this:

```
{ kind: 247, code: '\nexport as namespace React;' }
[!] Error: Could not load <rootProject>/node_modules/@types/react/index.d.ts (imported by <rootProject>/src/index.ts): unsupported node type
```

Seems like we're stuck with unsupported namespace issue again :(

**Comment by Swatinem** [comment]
> Could not load <rootProject>/node_modules/@types/react/index.d.ts

That should be easily solvable by defining `react` as `external`, which you should definitely do since you don’t want to bundle up all the types of an external library.

I will look into the other error tomorrow when I find some time.

**Comment by panjiesw** [comment]
Those error happened when I've already mark all modules in `node_modules` under `external` using this config:

```js
export default {
  // ....
  external: id => !id.startsWith('\0') && !id.startsWith('.') && !id.startsWith('/'),
  // ....
}
```

My generated bundle also contains only my source files, without `react` or any other external modules.

**Comment by Swatinem** [comment]
Sorry for the delay. This should be fixed in `v0.13.0`.
Here is a testcase: https://github.com/Swatinem/rollup-plugin-dts/blob/master/tests/testcases/react-components/index.ts

Please note that you really have to add `react` to the `external` list, also for the `dts` part of this plugin.

**Comment by panjiesw** [comment]
Thanks @Swatinem! Yeah I realized that we also need to configure external for the dts part
